ChangeLog.md

```diff
@@ -1,6 +1,22 @@
 
 # Change Log
 
+## Version 10.6.0
+
+### New features
+
+1. ``azcopy sync`` now supports the persistence of ACLs between supported resources (Windows and Azure Files) using the --persist-smb-permissions flag.
```

Comment:
Minor: sync does not support local <-> File, so it's for Azure Files <-> Azure Files only.